src/main/java/Backend/Drifty.java

```diff
@@ -61,21 +61,20 @@ public Drifty(String url, String downloadsDirectory, String fileNameOfTheDownloa
      */
     public void start() {
         Utility utility = new Utility(messageBroker);
-        boolean errorFlag = false;
         messageBroker.sendMessage("Validating the link...", DriftyConstants.LOGGER_INFO, "link");
         if (url.contains(" ")) {
             messageBroker.sendMessage("Link should not contain whitespace characters!", DriftyConstants.LOGGER_ERROR, "link");
-            errorFlag = true;
+            return;
```

Comment:
Suppresses the other messages when an error occurs. This allows the user to focus on the last message on the console which is the actual error.

Reply:
Ok @rohanshah18 👍, that's great 😃!